Makefile

```diff
@@ -276,7 +276,7 @@ ifdef SUITES
 SUITES_ARG = --suites $(SUITES)
 COMPLETE_SUITES_ARG = -args $(SUITES_ARG)
 endif
-TEST_FEATURE_GATES ?= WorkspaceMounts=true
+TEST_FEATURE_GATES ?= WorkspaceMounts=true,CacheAPIs=true,CacheAPIs=true
```

Comment:
Is there a reason that `CacheAPIs` is enabled twice?

Reply:
yes :D auto-rebase :D